src/lib/plugins/useItem.js

```diff
@@ -5,6 +5,7 @@ module.exports.player = function (player, serv, { version }) {
 
   player._client.on('block_place', ({ direction, location } = {}) => {
     const heldItem = player.inventory.slots[36 + player.heldItemSlot]
+    if(heldItem == undefined) return
```

Comment:
use ===
and add a space before the ( 
(or run standard --fix to fix these kind of things)